src/views/NotificationButton.tsx

```diff
@@ -124,13 +124,18 @@ class NotificationButton extends ComponentEx<IProps, IComponentState> {
       });
     }
 
+    const pendingActivities = notifications.filter(
+      (iter) => iter.type === 'activity' && iter.progress === undefined
+    );
+
     return (
       <div style={{ display: 'inline-block' }}>
         <Button id='notifications-button' onClick={this.toggle} ref={this.mButtonRef}>
           <Icon name='notifications' />
           <RadialProgress
             className='notifications-progress'
             data={combinedProgress}
+            spin={pendingActivities.length >= 1}
```

Comment:
As I said above: the spin flag now takes precedence over data, meaning that if we have notifications that have actual progress information we're not showing it.